### DISCUSSION TITLE
Custom style for components v. 4

### DISCUSSION THREAD

**Discussion by expdts** [taiga-family/taiga-ui#9542] [Q&A] [answered]
Обновился с версии 3.x до 4.11.0.
И теперь стили у всех компонентов (кнопки, чекбокс, ввод и т.п.) не применяются свои. Пробовал разные варианты, но ничего не получается. В документации изменение стилей не нашел.
Подскажите пожалуйста, как теперь можно изменить стиль у компонентов? [Пример](https://stackblitz.com/edit/angular-yc7jvr).

**Comment by waterplea** [comment, marked as answer]
Оно теперь не вложенное, а прямо на хосте. Поменяй на `button[data-appearance='orange']`. И `:disabled`

**Reply by expdts** [reply]
Спасибо, через присвоение класса тоже получилось `class="tui-button-orange"`.

**Reply by waterplea** [reply]
Ага, теперь нет вложенности и все стили висят на хосте, так что их можно переопределять просто классами, лишь бы приоритет стилей был выше.
